android/src/main/java/cl/json/social/FacebookStoriesShare.java

```diff
@@ -0,0 +1,114 @@
+package cl.json.social;
+
+import android.app.Activity;
+import android.content.ActivityNotFoundException;
+import android.content.Intent;
+import android.os.Environment;
+import android.net.Uri;
+import java.io.File;
+
+import cl.json.ShareFile;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReadableMap;
+
+/**
+ * Created by Kaio Duarte on 23-12-20.
+ */
+public class FacebookStoriesShare extends SingleShareIntent {
+
+    private static final String PACKAGE = "com.facebook.katana";
+    private static final String PLAY_STORE_LINK = "market://details?id=com.facebook.katana";
+
+    public FacebookStoriesShare(ReactApplicationContext reactContext) {
+        super(reactContext);
+        this.setIntent(new Intent("com.facebook.stories.ADD_TO_STORY"));
+    }
+
+    @Override
+    public void open(ReadableMap options) throws ActivityNotFoundException, IllegalArgumentException {
+        super.open(options);
+        this.shareStory(options);
+        // extra params here
+        this.openIntentChooser(options);
+    }
+
+    @Override
+    protected String getPackage() {
+        return PACKAGE;
+    }
+
+    @Override
+    protected String getDefaultWebLink() {
+        return null;
+    }
+
+    @Override
+    protected String getPlayStoreLink() {
+        return PLAY_STORE_LINK;
+    }
+
+    private void shareStory(ReadableMap options) {
+        if (!this.hasValidKey("appId", options)) {
+            throw new IllegalArgumentException("appId was not provided.");
+        }
+
+        if (!this.hasValidKey("backgroundImage", options) && !this.hasValidKey("backgroundVideo", options)
+                && !this.hasValidKey("stickerImage", options)) {
+            throw new IllegalArgumentException("Invalid background or sticker assets provided.");
+        }
+
+        Activity activity = this.reactContext.getCurrentActivity();
+
+        if (activity == null) {
+            TargetChosenReceiver.sendCallback(false, "Something went wrong");
+            return;
+        }
+
+        this.intent.putExtra("com.facebook.platform.extra.APPLICATION_ID", options.getString("appId"));
+        this.intent.putExtra("bottom_background_color", "#906df4");
+        this.intent.putExtra("top_background_color", "#837DF4");
```

Comment:
I think we already pass these params using the `options`, can we use these params here instead of using a hard-coded color?

Reply:
Actually there is no default value for options and these values are already hardcoded in the Objective-C code as well.

Reply:
I think we can go along with this PR, and add these options at another PR. I'm not familiar with FB Stories but it should be great to let these parameters being set.